### ISSUE TITLE
The text and layout need to be adjusted for better readability and usability across different screen sizes.

### ISSUE THREAD

**Issue by ashparshp** [layer5io/layer5#5640]
### Description
On smaller screens, the content appears cramped and misaligned, making it difficult to read and navigate.

#### Steps to Reproduce
Open the URL https://layer5.io/learn/service-mesh-labs/working-with-meshery-and-istio on a mobile device or resize your browser to a smaller screen width.

### Expected Behavior
- The page should be fully responsive and display well on all devices, including desktops, tablets, and smartphones.
- The text should be readable, and the layout should be user-friendly on smaller screens.

### Screenshots
![Card](https://github.com/layer5io/layer5/assets/150526286/c6a40cca-5bee-4664-9295-c54208908a26)

### Environment:
- Host OS:
- Browser:

---
<img src="https://raw.githubusercontent.com/layer5io/layer5/master/.github/assets/images/layer5/5-light-small.svg" width="24px" align="left" /><h2>Contributor Resources and <a href="https://layer5.io/community/handbook">Handbook</a></h2>

The layer5.io website uses Gatsby, React, and GitHub Pages. Site content is found under the [`master` branch](https://github.com/layer5io/layer5/tree/master).
- 📚 See [contributing instructions](https://github.com/layer5io/layer5/blob/master/CONTRIBUTING.md).
- 🎨 Wireframes and [designs for Layer5 site](https://www.figma.com/file/5ZwEkSJwUPitURD59YHMEN/Layer5-Designs) in Figma [(open invite)](https://www.figma.com/team_invite/redeem/qJy1c95qirjgWQODApilR9)
- 🙋🏾🙋🏼 Questions: [Discussion Forum](https://discuss.layer5.io) and [Community Slack](https://slack.layer5.io).

<img src="https://raw.githubusercontent.com/layer5io/layer5/master/.github/assets/images/buttons/community.webp" height="22px" align="left" />Join the Layer5 Community by submitting your [community member form](https://layer5.io/newcomer).

**Comment by pukarlamichhane** [comment]
@Ashparshp  sir can I work on this issue

**Comment by Ashparshp** [comment]
@pukarlamichhane Go ahead!

**Comment by Shubhf** [comment]
Hey @Ashparshp Can I take up this issue?

**Comment by saurabhkushwaha438** [comment]
i can't find where ,Working with Meshery and Istio section file is

**Comment by Ashparshp** [comment]
@jay270804, Maybe you can go ahead with it?

**Comment by jay270804** [comment]
@Ashparshp I will try to do my best on the issue !
Thanks for helping.

**Comment by Ashparshp** [comment]
@jay270804, Thanks!

**Comment by jay270804** [comment]
@Ashparshp When i tried to run the website for the first time using make setup and make site, the project worked perfectly.

But now when I am trying to run it again using the make site, it is getting stuck at "Getting queries from Components" - can you please give me a solution? I even cleared cache files but the issue is still prevailing.

**Comment by jay270804** [comment]
@Ashparshp Actually I recloned the code and then i tried to run it, that time the extraction was completed however when the website was compiling it failed.

Now, when I am again trying to run the website using 'make clean' or 'make site' it gets stuck here.
![Screenshot 2024-09-25 151252](https://github.com/user-attachments/assets/da46a7c0-2016-476b-947e-d083cebd3591)

**Comment by abdurrahmaan11265** [comment]
@Ashparshp Sir, can you please assign this issue to me

**Comment by Ashparshp** [comment]
@jay270804, were you able to set it up, or are you still working on it?

@abdurrahmaan11265, thanks for your interest. Let's see if @jay270804 is still working on it, and in the meantime, please check other open issues.

**Comment by jay270804** [comment]
Yes, I just got my ssd replaced as the previous one got corrupted. Now yesterday I was able to run it locally.

Ultimately, I am working on the issue.

**Comment by NahuelBarriga** [comment]
I'd love to help, I've got experience with front dev

**Comment by ashparshp** [comment]
Thank you @NahuelBarriga...

**Comment by jimmybatuhan** [comment]
I would like to give this a try if this is still open.

**Comment by NahuelBarriga** [comment]
Thanks for assigning this to me! I’ll start working on it and open a PR soon.

**Comment by ashparshp** [comment]
@jimmybatuhan Thanks! Since this has already been assigned to @NahuelBarriga, Please check other open issues as we proceed in chronological order.

**Comment by vr-varad** [comment]
@NahuelBarriga the issue still remains

![Image](https://github.com/user-attachments/assets/573a9f74-dc27-4385-921a-f4ac18ccc802)

**Comment by NahuelBarriga** [comment]
Hi @vr-varad, I've fixed the issue and submitted a PR following all the instructions, but it just hasn't been approved.

**Comment by vr-varad** [comment]
can u share the pr @NahuelBarriga

**Comment by NahuelBarriga** [comment]
Absolutelly, here's the link
https://github.com/layer5io/layer5/pull/6515

**Comment by HarK-github** [comment]
I can try this out @vr-varad , can you assign this to me.

**Comment by greedy-wudpeckr** [comment]
@vr-varad 
hello i want to try this , can u assign to me

**Comment by ARYANSHAH1567** [comment]
@HarK-github Are you still working on this issue?

**Comment by ravenloue** [comment]
@vr-varad - Is it possible to be assigned this since the last PR was closed in November due to inactivity?

**Comment by kishore08-07** [comment]
@ravenloue, could you confirm if this issue still persists?

**Comment by ravenloue** [comment]
@kishore08-07 - I just reviewed the pages that were involved; they've been changed from the "service-mesh-labs" collection to the "kanvas-labs" collection, and it appears that when this change was made, the visual bug was fixed. I believe this can be safely closed.

**Comment by Vanshikadahaliya** [comment]
I want to work on this issue.
